external/storm-kafka-monitor/src/main/java/org/apache/storm/kafka/monitor/KafkaOffsetLagUtil.java

```diff
@@ -373,16 +377,20 @@ private static Options buildOptions () {
         curatorFramework.start();
         String partitionPrefix = "partition_";
         String zkPath = oldKafkaSpoutOffsetQuery.getZkPath();
-        if (!zkPath.endsWith("/")) {
-            zkPath += "/";
+         if (zkPath.endsWith("/")) {
```

Comment:
I think it is better to check the variable before use it.  and the zkPath must not ends with `/`.
the `printUsageAndExit` method needs argument  `Options` which is not available in `getOldConsumerOffsetsFromZk`. so I use `exit` directly.